composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "php-amqp/php-amqp",
```

Comment:
Please verify vendor namespace & package name are acceptable

Reply:
Good point. The ext name itself is `amqp`, see https://github.com/php-amqp/php-amqp/blob/a9cbd81ea401d4ba41b72381008b331f5516823c/amqp.c#L386 so given the constrain on ext name in PIE tool itself (https://github.com/php/pie/blob/c6f0f4b2bfcc1de7e5f55b6317ef4d36b2ce49f2/src/ExtensionName.php#L26-L31), should it be named as `php-amqp/amqp` then ? And does PIE impose limitations on what vendor name could be?

Reply:
The *Package Name* does not have any constraints. But the *Extension Name* always has done. 

I have defined the *Extension Name* according to the existing `package.xml` as `amqp` on L36, so the *Package Name* can really be anything that makes sense for the project; same with the vendor 👍

Reply:
Got you, thanks for clarification.

*(might be unrelated to the scope of PR work)* Interesting enough, the composer package is still pointing to the old repo name - `pdezwart/php-amqp`. And the reason we got composer.json is to be able to fetch stub files locally, it was the days before PHPStorm had it bundled and I guess the PHPStorm itself was not that widely adopted itself, afair

Reply:
Not an issue at all, we would just need to set up a redirect from that old one to a new one, afair there is sth like that in packagist. We would likely need Lars or Pieter for that though.